pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -378,8 +378,8 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 				return
 			}
 		}
-		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by %v/%v on node %v",
-			pod.Namespace, pod.Name, c.Name())
+		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by a pod scheduled by %s on node %v",
+			pod.Namespace, pod.Name, pod.Spec.SchedulerName, c.Name())
```

Comment:
pod's namespace/name should be removed:

```suggestion
			pod.Spec.SchedulerName, c.Name())
```

Reply:
That's right, fix pushed